### ISSUE TITLE
Remove outdated calendars.

### ISSUE THREAD

**Issue by guusdk** [xsf/xmpp.org#280]
The website serves out-of-date/unused iCal files, including (but perhaps not limited to):
* http://xmpp.org/calendar/xsf-council.ics
* http://xmpp.org/calendar/team-communication.ics 
* http://xmpp.org/calendar/team-techreview.ics
* http://xmpp.org/calendar/team-editor.ics

To avoid confusion, these should be removed - as can the infrastructure that was used to create/populate them.

Please add a permanent redirection from all of these URLs to https://calendar.google.com/calendar/ical/64v3vs15qlalgqv0j7r99ikm1c%40group.calendar.google.com/public/basic.ics - which is the new shared calendar instance that is currently in use.

**Comment by tfar** [comment]
Thanks. Will take a look at this this evening.

**Comment by horazont** [comment]
Please use a temporary redirect. Permenant redirects are hard to get rid of, which we may want to at some point.

**Comment by intosi** [comment]
I'm pretty sure redirects from those topic specific calendars to a general, covers-all-topics, will annoy current subscribers, especially because they didn't generate for them events recently.  I propose not adding the redirects.

**Comment by guusdk** [comment]
I'm fine with having no redirects, but let's at least remove the old, out-of-date, unmaintained calendars.

**Comment by intosi** [comment]
For internal reference, these are generated by the old website generator, at /home/xsf/xmpp-hg/calgen.py.

What do we want to happen to /calendar/xsf-extensions.ics and /calendar/xsf-all.ics, both of which seem to be more or less up-to-date?  Do we want to migrate those to tooling integrated in the new site?

**Comment by guusdk** [comment]
If they work as intended, I'd leave them be. My remark regarding removing the infrastructure that generates the iCal files was based on the assumption that it was not used anymore.

**Comment by guusdk** [comment]
This relates to another issue that @bear raised many moons ago: #108

**Comment by Kev** [comment]
On 24 Mar 2017, at 09:32, Edwin Mons <notifications@github.com> wrote:
> 
> For internal reference, these are generated by the old website generator, at /home/xsf/xmpp-hg/calgen.py.
> 
> What do we want to happen to /calendar/xsf-extensions.ics and /calendar/xsf-all.ics, both of which seem to be more or less up-to-date? Do we want to migrate those to tooling integrated in the new site?

I wonder how hard it’d be to have the script generate them from the google calendar?

/K

**Comment by tfar** [comment]
If google calendar provides a way to tag the events so we can distinguish what should go in what calendar it shouldn't be that hard.

**Comment by guusdk** [comment]
Perhaps it is, but is it worth the effort of building and maintaining it? I'd go for lean and mean, and improve only after someone expressed interest. After all, the old calendars were not used for at least a couple of years.

**Comment by guusdk** [comment]
This issue was 'fixed' by the server loss (the rebuild does not include the old calendars). I'm closing this issue. If someone would like to see redirects, or other restoration work, please re-open, or create a new issue.
